docs/release-notes/3.1.0.md

```diff
@@ -1,3 +1,4 @@
 # Bug Fixes
 
-- Fix behaviour of in-cluster connectivity when using JWT handler in APIRule. Now, in-cluster connectivity is blocked. [#1823](https://github.com/kyma-project/api-gateway/pull/1823)
\ No newline at end of file
+- Fix behaviour of in-cluster connectivity when using JWT handler in APIRule. Now, in-cluster connectivity is blocked. [#1823](https://github.com/kyma-project/api-gateway/pull/1823)
```

Comment:
please change RN to 3.0.3

Reply:
Those will be added with cherrypick, we don't hold rn for patches on main